extensions/binding/org.eclipse.smarthome.binding.astro/OSGI-INF/AstroDiscoveryService.xml

```diff
@@ -9,7 +9,7 @@
     http://www.eclipse.org/legal/epl-v10.html
 
 -->
-<scr:component xmlns:scr="http://www.osgi.org/xmlns/scr/v1.1.0" immediate="true" name="org.eclipse.smarthome.binding.astro.discovery">
+<scr:component xmlns:scr="http://www.osgi.org/xmlns/scr/v1.1.0" immediate="false" name="org.eclipse.smarthome.binding.astro.discovery">
```

Comment:
I'd agree with @afuechsel - we usually start all components of which we know that we need them immediately.
Also, this change is unrelated to the topic of this PR, so it would be better to revert it.